.env

```diff
@@ -0,0 +1,5 @@
+SERVER_PUBLIC_DOMAIN="http://didcomm-mediator.com"
```

Comment:
Have you checked if this domain name is free?

Reply:
untill he have a valid name we can put anything valid here for now that why we the value comes from and env file which is customisable